spec/function.dd

```diff
@@ -1026,18 +1026,23 @@ class Bar : Foo
 ------
 )
 
-        $(P Virtual functions all have a hidden parameter called the
-        $(I this) reference, which refers to the class object instance for which
-        the function is called.
+        $(P 
+            Non-static (i.e., virtual functions and $(D final) member
+            functions) all have a hidden parameter called the
+            $(I this) reference, which refers to the class object for which
+            the function is called.
         )
 
         $(P
             Functions with `Objective-C` linkage have an additional hidden,
             unnamed, parameter which is the selector it was called with.
         )
 
-        $(P To directly call a base member function, insert
-        base class name before the member function name. For example:
+        $(P To directly call a base member function (i.e., avoid dynamic
+        binding on member function call), insert base class name before the
+        member function name. Note that when when invoked in this way, the
+        member function will not be passed the hidden $(I this) parameter.
+        For example:
```

Comment:
I think you're getting slightly confused here as to what the `statically` is referring to in the example below. `static` here means calling the function directly i.e. not calling through the function pointer int the virtual function table. This should not be confused with static member functions, which don't take a `this` reference.

think about what would happen if `B` were instead 
```
class B
{
    int a;
    int foo() { return a; }
}
```
and you tried to call `B.foo()` without the `this` parameter.

Reply:
@thewilsonator this is corrected; it was a merging error when re-basing my branch as I removed this in response to your comment in 2019 ;)

https://github.com/dlang/dlang.org/pull/2570/commits/380da3c29eeeba5c4f7610daa81f4f6ce6a36c02